requirements.txt

```diff
@@ -39,11 +43,12 @@ pyparsing==2.2.0
 python-dateutil==2.5.3
 python-env==1.0.0
 pytz==2017.2
-requests==2.13.0
+requests==2.16.5
 simplegeneric==0.8.1
 singledispatch==3.4.0.3
 six==1.10.0
 static3==0.7.0
 traitlets==4.2.1
 typing==3.6.1
+urllib3==1.21.1
 whitenoise==3.3.0
```

Comment:
I think it would be best to squash all your commits into one to make it look cleaner as you currently have merge commits pulled into yours.